File: src/main/java/com/mamaliang/npps/MainVerticle.java
package com.mamaliang.npps;

import com.mamaliang.npps.objects.RedisClientSingleton;
import io.vertx.core.AbstractVerticle;

/**
 * @author gaof
 * @date 2024/8/30
 */
public class MainVerticle extends AbstractVerticle {

  @Override
  public void start() {
    RedisClientSingleton.initialize(vertx);
    vertx.deployVerticle("com.mamaliang.npps.cas.server.TgtCoroutineVerticle");
  }


}

File: src/main/kotlin/com/mamaliang/npps/MainVerticle.kt
package com.mamaliang.npps

import com.mamaliang.npps.cas.server.verticle.CasServerVerticle
import com.mamaliang.npps.common.RedisClient
import io.vertx.kotlin.coroutines.CoroutineVerticle
import io.vertx.kotlin.coroutines.coAwait

class MainVerticle : CoroutineVerticle() {

    override suspend fun start() {
        RedisClient.initialize(vertx)
        vertx.deployVerticle(CasServerVerticle::class.java.name).coAwait()
    }

}




